Mathlib/Topology/Order/Basic.lean

```diff
@@ -2832,6 +2832,14 @@ theorem IsClosed.csInf_mem {s : Set α} (hc : IsClosed s) (hs : s.Nonempty) (B :
   (isGLB_csInf hs B).mem_of_isClosed hs hc
 #align is_closed.cInf_mem IsClosed.csInf_mem
 
+theorem IsClosed.isLeast_csInf {s : Set α} (hc : IsClosed s) (hs : s.Nonempty) (B : BddBelow s) :
```

Comment:
Does this follow from `IsCompact.isLeast_sInf` or vice versa?

Reply:
It can't follow from `IsCompact.*` becase a closed set doesn't have to be compact. For the other implication, the TC assumptions are different. I'm not sure if we can sync them but definitely not in this PR.